### DISCUSSION TITLE
Edu Mail Rejection

### DISCUSSION THREAD

**Discussion by NewVu2** [community/community#129823] [GitHub Education]
### Hi everyone,

I'm a student at **Virtual University of Pakistan** and I verify my account but at the time of upload image it only opens my front camera how I can send my university card image from my front camera. There is no option of upload the image from my mobile files please help??????????????

**Comment by NewVu2** [comment]
And 2nd thing on my card the expire date is on back how I can upload this please guide me.

**Reply by Kissada-Waravit** [reply]
I received an email response for a similar reason. The team's advice states:
You may use multiple documents in your image. As long as the documents are legible

I still cannot submit documents through the account because my account is new. And my student card doesn't have a date. Which is very sad.

**Comment by Yodha-Sudarsi** [comment]
@NewVu2 , You can also upload university fee receipt that will also help to approve your application. My College ID has no pass out year on that. They rejected it for the first time. 
Then I uploaded both my ID card and **College fee receipt**. Later they approved.
 _Upload multiple documents that proves you are a student._

**Reply by Kissada-Waravit** [reply]
@Yodha-Sudarsi Okay, I'll give it a shot, but I'm a bit concerned about a potential issue. You see, all my official university documents, like my transcripts and enrollment forms, have my full legal name written in Thai. However, my username for the university's online systems, including the student portal and email, is in English. I'm worried this discrepancy might cause problems when trying to access or link my accounts.
